cloudinary_cli/modules/regen_derived.py

```diff
@@ -63,6 +62,9 @@ def regen_derived(trans_str, eager_notification_url,
                                        api_name="admin",
                                        auto_paginate=auto_paginate,
                                        force=force, return_data=True)
+    if trans_details is False:
+        exit()
```

Comment:
Maybe write an error message saying what went wrong? Just exiting can be very confusing. And it you run it in script/cron job - you have no idea what happened/ did not happen.

Reply:
as discussed I've updated this to `return False` instead